crates/chainio/clients/fireblocks/src/status.rs

```diff
@@ -33,3 +35,26 @@ pub enum Status {
     #[serde(rename = "FAILED")]
     Failed,
 }
+
+impl Status {
+    pub fn as_str(&self) -> &'static str {
+        match self {
+            Status::Approved => "Approved",
```

Comment:
I think there is a mismatch here. In the serilization with serde, the string is in capital letters, but in here it has only the first letter in capital.